subsys/event_manager/event_manager_priv.h

```diff
@@ -315,23 +317,21 @@ struct event_type {
 	/** Pointer to the element directly after the array of subscribers. */
 	const struct event_subscriber	*subs_stop;
 
-	/** Bool indicating if the event is logged by default. */
-	bool init_log_enable;
-
 	/** Function to log data from this event. */
 	int (*log_event)(const struct event_header *eh, char *buf,
 			      size_t buf_len);
 
 	/** Custom data related to tracking. */
 	const void *trace_data;
 
+	/** Array of flags dedicated to event type. */
+	const uint8_t event_type_flags;
+
 #if IS_ENABLED(CONFIG_EVENT_MANAGER_PROVIDE_EVENT_SIZE)
 	/** The size of the event structure */
 	uint16_t struct_size;
-
-	/** The flag that stores the information if the event type contains dyndata */
-	bool has_dyndata;
 #endif
+
```

Comment:
don't add this empty line. the same above with line 329.
try to limit change - simplifies the review